src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -300,25 +309,17 @@ private void ScanMultiLineRawStringLiteral(ref TokenInfo info, int startingQuote
             currentLineWhitespace.Clear();
             ConsumeWhitespace(currentLineWhitespace);
 
-            if (SyntaxFacts.IsNewLine(TextWindow.PeekChar()))
```

Comment:
this just mirrors the logic in the interpolation code (where i took an approach with less repitition and hopefully more clarity).